frontend/src/component/admin/apiToken/ConfirmToken/UserToken/UserToken.tsx

```diff
@@ -24,25 +24,24 @@ export const UserToken = ({ token }: IUserTokenProps) => {
     };
 
     return (
-        <div
-            style={{
-                backgroundColor: '#efefef',
-                padding: '2rem',
-                borderRadius: '3px',
-                margin: '0',
-                marginTop: '1rem',
+        <Box
+            sx={theme => ({
+                backgroundColor: theme.palette.background.elevation2,
+                padding: theme.spacing(4),
+                borderRadius: `${theme.shape.borderRadius}px`,
```

Comment:
Could this one be simplified like this?

```suggestion
                borderRadius: theme.shape.borderRadius,
```